### ISSUE TITLE
Missing Labels for login, register, and recovery forms

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1163]
On page https://cryptpad.fr/login/ the input fields miss a label (both visually and in the code). The placeholders cannot function as labels, because they disappear when a user starts to type.  

The input field that appears when clicking on ‘Add link to website’ https://cryptpad.fr/profile/ visually misses label, as well as in the code.

**Comment by dariiing** [comment]
<img width="553" alt="image" src="https://github.com/cryptpad/cryptpad/assets/95225431/b785b1f9-d971-4779-8711-2b368bc3ce4e">

**Comment by dariiing** [comment]
<img width="462" alt="image" src="https://github.com/cryptpad/cryptpad/assets/95225431/df22d30c-b226-4c90-ac80-83e028c30c21">

**Comment by dariiing** [comment]
<img width="560" alt="image" src="https://github.com/cryptpad/cryptpad/assets/95225431/f4a2a239-1cad-4f71-9c0e-fb85a154d794">
